scripts/buildNumberIncrement.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+HOCKEYAPP_ID=$1
+HOCKEYAPP_KEY=$2
+INFO_PLIST_PATH=$3
+
+hockeyLastBuildNumber=$(curl -H "X-HockeyAppToken: $HOCKEYAPP_KEY" https://rink.hockeyapp.net/api/2/apps/$HOCKEYAPP_ID/app_versions | python -c "import sys, json; print json.load(sys.stdin)['app_versions'][0]['version']")
```

Comment:
Why not using the buildnumber from jenkins here? The hockeyapp build number and the jenkins build number could be the same.

Reply:
It's not a bad idea. But does that mean that the build number will not reset after the app version is incremented (eg. 1.0 (123) to 1.1 (124) as oppose to 1.1 (1))?

Reply:
no the build number will not reset(only if some spanish gringo deletes all configurations in jenkins). But why reseting the build number anyway, don't you want to have unique build numbers to identify all the builds?

Reply:
Yeah, you're right. Changed it.